src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_RENT = 40;
+  const LONG_TEMR = 7;
+  const SHORT_TERM = 3;
+  const TOTAL_PRICE = days * PRICE_RENT;
+
+  const LONG_TEMR_DISCOUNT = 50;
```

Comment:
There is a typo in the constant name `LONG_TEMR_DISCOUNT`. It should be `LONG_TERM_DISCOUNT` to maintain consistency and avoid confusion.